BUILD_macos.md

```diff
@@ -70,6 +70,11 @@ PRs with corrections and updates are welcome!
   brew install libfido2
   ```
 
+* To install `yarn` for building the UI
+  * Install Node.js from the [Node.js Releases/Download Page](https://nodejs.org/en/download/)
```

Comment:
Currently we require `v16` but within the next month we're planning to upgrade to `v18`.

Reply:
I'll add a comment about it.